readme.md

```diff
@@ -342,6 +342,12 @@ As in: OOPs I used Java!
 
 ---
 
+**Q:** Why do C programmers write their letters in *bold*?
+
+**A:** Because they're *strongly* typed.
```

Comment:
I don't get it

Reply:
C is a strongly typed language. Strong is the tag used to write bold.

Reply:
The single `*` is italic, maybe you meant to use `**` for bold?